certbot-auto

```diff
@@ -131,7 +131,7 @@ su_sudo() {
 
 SUDO_ENV=""
 export CERTBOT_AUTO="$0"
-if [ -n "${LE_AUTO_SUDO+x}" ]; then
```

Comment:
Is there a reason this isn't defined via a command-line argument? I'm very curious now, and resisting the urge to try to 'improve' things.